content/docs/command-reference/checkout.md

```diff
@@ -10,17 +10,17 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
                     [--relink] [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these stages or .dvc files.
-                 Using -R, directories to search for stages or .dvc
-                 files can also be given.
+  targets       Limit command scope to these tracked files/directories,
+                .dvc files, or stage names.
 ```
 
 ## Description
 
-`.dvc` and `dvc.lock` [files](/doc/user-guide/dvc-files-and-directories) act as
-pointers to specific version of data files or directories tracked by DVC. This
-command synchronizes the workspace data with the versions specified in the
-current `.dvc` and `dvc.lock` files.
+[`dvc.lock`](/doc/user-guide/dvc-files-and-directories#dvclock-file) and
+[`.dvc`](/doc/user-guide/dvc-files-and-directories#dvc-files) files act as
+pointers to the <abbr>cached</abbr> contents of data tracked by DVC. This
```

Comment:
it can be a bit confusing - pointers to contents vs pointers to cached files. Second is a bit more explicit, even though we expose more implementation details.

Reply:
Agree. Also the manual links are no longer needed. Updated as part of a36ba41f.